lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -170,31 +170,50 @@ def self.move_after_record_with_position(records, targeted_record_position)
       # lower in the list of all chapters. Likewise, <tt>chapter.first?</tt> would return +true+ if that chapter is
       # the first in the list of all chapters.
       module InstanceMethods
+        # https://github.com/brendon/acts_as_list/commit/75ec5b9b3a7e72a35bc5bd95ca9061d44ece2a76
+        def position_before_save_changed?
+          if ActiveRecord::VERSION::MAJOR >= 5 && ActiveRecord::VERSION::MINOR >= 1
+            saved_change_to_attribute? position_column
+          else
+            send "#{position_column}_changed?"
+          end
+        end
+        # https://github.com/brendon/acts_as_list/commit/64b6e15200f1e8eddfbeedf35a225258585b13b4
+        def position_before_save
+          if ActiveRecord::VERSION::MAJOR >= 5 && ActiveRecord::VERSION::MINOR >= 1
```

Comment:
what happens at version 6.0?

Reply:
i have a feature open to address this. i think we should fork the link above and repatch what we changed to be honest.

Reply:
```suggestion
          if ActiveRecord::VERSION::MAJOR >= 6 || (ActiveRecord::VERSION::MAJOR >= 5 && ActiveRecord::VERSION::MINOR >= 1)
```
or did you mean something like this?